test

```diff
@@ -83,6 +83,16 @@ docker run -d --name=jupyter_test_r --read-only --net=none \
 sleep 3
 docker kill jupyter_test_r && docker rm jupyter_test_r
 
+# Check that papermill within python is installed.
+docker run -d --name=papermill_test_r --read-only --net=none \
+    -e HOME=/tmp \
+    -v $PWD:/input:ro -v /tmp/rstats-build/working:/working \
+    -v /tmp/rstats-build/tmp:/tmp -v /tmp/rstats-build/devshm:/dev/shm \
+    -w=/working \
+    "$IMAGE_TAG" python -c 'import sys;import papermill as pm; print(pm.__version__)' --allow-root --ip="*"
+sleep 3
+docker kill papermill_test_r && docker rm papermill_test_r
```

Comment:
Your docker run command can take --rm so that it deletes it automatically when it stops it.

Reply:
This won't fail if papermill is not installed. The command above it succeeds because jupyter notebook is a long-lived command and docker kill expects to kill it, and will fail if jupyter notebook didn't boot and stay booted.

In this case though, you're just running a python command which should exit on its own. I think you may need to actually check the exit code, or the output of the container to verify that papermill really is working.

Reply:
Thanks, good call!